src/sinks/util/udp.rs

```diff
@@ -223,6 +226,19 @@ impl tower::Service<BytesMut> for UdpService {
             // TODO: Add reconnect support as TCP/Unix?
             let result = udp_send(&mut socket, &msg).await.context(SendSnafu);
             let _ = sender.send(socket);
+
+            if result.is_ok() {
+                // TODO: This is obviously not happening before things like compression, etc, so it's currently a
```

Comment:
Not so much a TODO here as a note or warning.

Reply:
Fair point.

Reply:
Addressed in 6b8a34b9f.